build.psm1

```diff
@@ -1182,17 +1196,17 @@ function Start-PSPackage {
         "appx" {
             $Arguments = @{
                 PackageNameSuffix = $NameSuffix
-                PackageVersion = $Version;
-                SourcePath = $Source;
+                PackageVersion = $Version
+                PackageSourcePath = $Source
                 AssetsPath = "$PSScriptRoot\assets"
             }
             New-AppxPackage @Arguments
         }
         default {
             $Arguments = @{
-                Type = $_;
-                Source = $Source;
-                Name = $Name;
+                Type = $_
+                PackageSourcePath = $Source
```

Comment:
Renamed this parameter to PackageSourcePath in the underlying function for consistency